intranet/templates/emailfwd/senior_forward.html

```diff
@@ -30,7 +30,7 @@ <h2>
         Senior Email Forwarding
     </h2>
 
-    <p><b>By the end of the summer after graduation, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the summer, this address can't be changed.</p>
+    <p><b>At the conclusion of the school year, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the summer, this address can't be changed.</p>
```

Comment:
```suggestion
    <p><b>At the conclusion of the school year, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the school year, this address can't be changed.</p>
```

Reply:
Hi! I just changed it using your recommendations. Please let me know if I need to change anything else.